perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ModulesUtilsBlImpl.java

```diff
@@ -579,7 +610,30 @@ public void checkFormatOfShell(String shell, Attribute attribute) throws WrongAt
 		if (!match.matches()) {
 			throw new WrongAttributeValueException(attribute, "Bad shell attribute format " + shell);
 		}
+	}
 
+	public void checkAttributeRegex(Attribute attribute, String defaultRegex) throws InternalErrorException, WrongAttributeValueException {
+		if (attribute == null || attribute.getValue() == null) throw new InternalErrorException("Attribute or it's value is null.");
+		String attributeValue = (String) attribute.getValue();
+		try {
+			String regex = BeansUtils.getPropertyFromCustomConfiguration("perun-namespaces.properties", attribute.getFriendlyName() + ":regex");
```

Comment:
This is the same problem like with logins and group names. We don't want to read the file for every time we call this method. Better will be fill or use perunNamespaces map where this value should be too.